decomp2d/fft_fftw3.f90

```diff
@@ -5,20 +5,20 @@
 ! decomposition. It also implements a highly scalable distributed
 ! three-dimensional Fast Fourier Transform (FFT).
 !
-! Copyright (C) 2009-2021 Ning Li, the Numerical Algorithms Group (NAG)
+! Copyright (C) 2009-2011 Ning Li, the Numerical Algorithms Group (NAG)
```

Comment:
?

Reply:
We can cancel these pull requests.I’ll put them back when finish. I am just testing stuffs with Semih